tests/mlx5_base.py

```diff
@@ -411,11 +412,15 @@ def traffic_with_bad_flow(client, server, iters, gid_idx, port):
                 u.send(client, c_send_object, send_op, True, qp_idx,
                        ah_client, False)
                 try:
-                    u.poll_cq(client.cq)
+                    wcs = u._poll_cq(client.cq)
                 except PyverbsError as ex:
-                    if client.bad_flow_handling(qp_idx, ex, True):
+                    if client.bad_flow_handling(qp_idx, e.IBV_WC_SUCCESS, True):
                         continue
                     raise ex
+                if wcs[0].status != e.IBV_WC_SUCCESS:
```

Comment:
shouldn't his check replace line 417?

Reply:
```
414                 try:                                                                                                                                                                                                               
415                     wcs = u._poll_cq(client.cq)                                 
416                 except PyverbsError as ex:                                      
417                     if client.bad_flow_handling(qp_idx, e.IBV_WC_SUCCESS, True):
418                         continue                                                
419                     raise ex                                                    
420                 if wcs[0].status != e.IBV_WC_SUCCESS:                           
421                     if client.bad_flow_handling(qp_idx, wcs[0].status, True):   
422                         continue   
```
In my understanding, line 417 is in a try...except block, in where it cannot get a correct wcs.

Reply:
Oh, I see what you did there. you can leave it as is.